roles/mesos/tasks/follower.yml

```diff
@@ -78,6 +78,15 @@
   tags:
     - mesos
 
+- name: set follower resources
+  sudo: yes
+  copy:
+    dest: /etc/mesos-slave/resources
+    content: "{{ mesos_resources }}"
+  notify: restart mesos follower
```

Comment:
for now we might want to remove the file that `mesos-slave` suggests when it starts up and sees a conflict, instead of the entire meta folder. I think recovery data is stored in the meta directory too. @stevendborrelli, :thumbsup:?